.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
     - php: hhvm
       env: DB=mysqli
   allow_failures:
-    - php: 7.0
+    - env: DB=sqlite3
```

Comment:
As I mentioned on IRC... sqlite3 is incompatible (always has been) with this extension.